### ISSUE TITLE
Throwing error immediately after installation

### ISSUE THREAD

**Issue by Shujee** [mtxr/vscode-sqltools#1038]
Getting the following error in the Terminal immediately after installing the extension:

```
$ node -e 'require(\"fs\").writeFileSync(\"C:\\Users\\<USERNAME>\\AppData\\Local\\vscode-sqltools\\Data\\.node-runtime\", process.exec
Path)' && exit 0
[eval]:1
require(\"fs\").writeFileSync(\"C:\\Users\\<USERNAME>\\AppData\\Local\\vscode-sqltools\\Data\\.node-runtime\", process.execPath)
        ^

SyntaxError: Invalid or unexpected token
←[90m    at new Script (node:vm:100:7)←[39m
←[90m    at createScript (node:vm:257:10)←[39m
←[90m    at Object.runInThisContext (node:vm:305:10)←[39m
←[90m    at node:internal/process/execution:75:19←[39m
    at [eval]-wrapper:6:22
←[90m    at evalScript (node:internal/process/execution:74:60)←[39m
←[90m    at node:internal/main/eval_string:27:3←[39m

```

**Desktop (please complete the following information):**
 - SQLTools Version: 0.4.1
 - VS Code Version: 1.73.1
 - OS: Windows 11
 - Driver:
   - [ ] MySQL/MariaDB

**Additional context**
Reinstalling the extension and/or restarting VSCode doesn't help. Creating/removing the .node-runtime folder doesn't help either. Shows this message when VSCode starts:

![image](https://user-images.githubusercontent.com/5603658/202076985-ca46c534-3910-40c4-9e7c-a304f4362348.png)

**Comment by gjsjohnmurray** [comment]
On the dropdown on the Terminal tab of the Panel, which shell is marked as your default?

**Comment by Shujee** [comment]
Git Bash

**Comment by gjsjohnmurray** [comment]
The algorithm for detecting the path of a node runtime isn't handling the case where a Windows instance of VS Code is using Git Bash as the default shell.

@Shujee a workaround for you is to edit your user-lever settings.json file and look for `sqltools.useNodeRuntime`. If you find it, delete the line. If you still get the problem, check the file `.vscode/settings.json` in the workspace you opened in VS Code, and remove that setting from it.

Knowledge of where your node runtime is located is only required if you are using the SQLite driver.

**Comment by Shujee** [comment]
Thanks @gjsjohnmurray . Fixed it for me.

**Comment by gjsjohnmurray** [comment]
Resolved by #1059
